compiler/tools/fpp-to-cpp/test/array/PrimitiveU16ArrayAc.ref.cpp

```diff
@@ -0,0 +1,208 @@
+// ======================================================================
+// \title  PrimitiveU16ArrayAc.cpp
+// \author Generated by fpp-to-cpp
+// \brief  cpp file for PrimitiveU16 array
+// ======================================================================
+
+#include <cstring>
+#include <cstdio>
+#include <cinttypes>
+
+#include "Fw/Types/Assert.hpp"
+#include "Fw/Types/StringUtils.hpp"
+
+#include "PrimitiveU16ArrayAc.hpp"
+
+namespace M {
+
+  // ----------------------------------------------------------------------
+  // Constructors
+  // ----------------------------------------------------------------------
+
+  PrimitiveU16 ::
+    PrimitiveU16() :
+      Serializable()
+  {
+    // Construct using element-wise constructor
+    *this = PrimitiveU16(
+      1,
+      2,
+      3
+    );
+  }
+
+  PrimitiveU16 ::
+    PrimitiveU16(const ElementType (&a)[SIZE]) :
+      Serializable()
+  {
+    for (U32 index = 0; index < SIZE; index++) {
+      this->elements[index] = a[index];
+    }
+  }
+
+  PrimitiveU16 ::
+    PrimitiveU16(const ElementType& e) :
+      Serializable()
+  {
+    for (U32 index = 0; index < SIZE; index++) {
+      this->elements[index] = e;
+    }
+  }
+
+  PrimitiveU16 ::
+    PrimitiveU16(
+        const ElementType (&e1),
```

Comment:
The parentheses around (&e1) are unusual. Can we write `const ElementType& e1` as elsewhere?

Reply:
Ok, I changed it and updated the unit tests.

Reply:
Looks good!